fusion-plugin-react-router/flow-typed/npm/history_v4.9.x.js

```diff
@@ -151,5 +123,19 @@ declare module 'history/createHashHistory' {
     ) => void,
   };
 
-  declare export default (opts?: HistoryOpts) => HashHistory;
+  declare function createHashHistory(opts?: HashHistoryOpts): HashHistory;
+
+  declare type LocationType = {
```

Comment:
Most of the updates are pulled from flow-typed.  We had made some changes to these in the past so they were not perfectly synchronized.  For example, `parsePath` and `createPath` are used in our package but do not exist in the flow-typed libdefs.